packages/react/src/components/Table/index.js

```diff
@@ -0,0 +1,158 @@
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { useTable, useSortBy } from "react-table";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+const HeaderGroup = ({ className, children, ...props }) => (
+  <tr className={cx("Table-HeaderGroup", className)} {...props}>
+    {children}
+  </tr>
+);
+
+HeaderGroup.propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node,
+};
+
+HeaderGroup.defaultProps = {
+  className: undefined,
+  children: undefined,
+};
+
+const HeaderCell = ({ className, children, isSorted, ...props }) => (
+  <th
+    className={cx("Table-HeaderCell", className, {
+      "is-sorted": isSorted,
+    })}
+    {...props}
+  >
+    {children}
+  </th>
+);
+
+HeaderCell.propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node,
+  isSorted: PropTypes.bool,
+};
+
+HeaderCell.defaultProps = {
+  className: undefined,
+  children: undefined,
+  isSorted: false,
+};
+
+const RowGroup = ({ className, children, ...props }) => (
+  <tr className={cx("Table-RowGroup", className)} {...props}>
+    {children}
+  </tr>
+);
+
+RowGroup.propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node,
+};
+
+RowGroup.defaultProps = {
+  className: undefined,
+  children: undefined,
+};
+
+const RowCell = ({ className, children, ...props }) => (
+  <td className={cx("Table-RowCell", className)} {...props}>
+    {children}
+  </td>
+);
+
+RowCell.propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node,
+};
+
+RowCell.defaultProps = {
+  className: undefined,
+  children: undefined,
+};
+
+const Table = ({ className, columns, data, isSortable }) => {
+  const { headerGroups, rows, prepareRow } = useTable(
+    {
+      columns,
+      data,
+    },
+    useSortBy
+  );
+
+  return (
+    <table className={cx("Table", className)}>
+      <thead>
+        {headerGroups.map(({ getHeaderGroupProps, headers }) => (
+          <HeaderGroup {...getHeaderGroupProps()}>
+            {headers.map(
+              ({
+                getHeaderProps,
+                getSortByToggleProps,
+                isSorted,
+                isSortedDesc,
+                render,
+              }) => (
+                <HeaderCell
+                  {...getHeaderProps(isSortable && getSortByToggleProps())}
+                  isSorted={isSorted}
+                >
+                  <Text>
+                    {render("Header")}
+                    {isSorted && isSortedDesc && <Icon icon="IconArrowUp" />}
+                    {isSorted && !isSortedDesc && <Icon icon="IconArrowDown" />}
+                    {isSortable && !isSorted && (
+                      <Icon icon="IconChevronUpDown" />
+                    )}
+                  </Text>
+                </HeaderCell>
+              )
+            )}
+          </HeaderGroup>
+        ))}
+      </thead>
+
+      <tbody>
+        {rows.map(row => (
+          <RowGroup {...prepareRow(row)} {...row.getRowProps()}>
```

Comment:
```suggestion
          <RowGroup {...getRowProps(row)} {...row.getRowProps()}>
```

🌷
does this sound like a better name?

Reply:
b/c you have `getCellProps`

Reply:
`prepareRow` is pulled from the main hook 😉 
```
const { headerGroups, rows, prepareRow } = useTable(
    {
      columns,
      data,
    },
    useSortBy
  );
```

Reply:
How about
```
const { headerGroups, rows, prepareRow: getRowProps } = useTable(
```
then?

Like, I'm not insisting, but for the consistency POV, could be nice.
Dunno, up to you to decide

Reply:
`prepareRow` doesn't return any props, it actually "prepares" rows to be rendered 🙂

Reply:
I moved this part higher in the component so it's not confusing 👍